chainer/functions/math/det.py

```diff
@@ -0,0 +1,106 @@
+import numpy.linalg
+
+from chainer import cuda
+from chainer import function
+from chainer.functions.array.reshape import Reshape
+from chainer.functions.math.inv import _inv_gpu
+from chainer.functions.math.matmul import _as_batch_mat
+from chainer.functions.math.matmul import _check_ndim
+from chainer.functions.math.matmul import _convert_type
+from chainer.functions.math.matmul import _get_ld
+from chainer.functions.math.matmul import _mat_ptrs
+from chainer import utils
+
+
+def _det_gpu(b):
```

Comment:
Do you think we should add this function to CuPy, although NumPy does not have corresponding method?